homeassistant/components/climate/__init__.py

```diff
@@ -498,7 +498,7 @@ async def async_turn_on(self) -> None:
         """Turn the entity on."""
         if hasattr(self, "turn_on"):
             # pylint: disable=no-member
-            await self.hass.async_add_executor_job(self.turn_on)
+            await self.hass.async_add_executor_job(self.turn_on)  # type: ignore
```

Comment:
Please ignore specific errors, e.g. `# type: ignore[no-any-return]`

Reply:
Thanks, fixed in 28540f7